support-workers/src/main/scala/com/gu/emailservices/DigitalPackEmailFields.scala

```diff
@@ -2,106 +2,81 @@ package com.gu.emailservices
-            "City" -> "", //TODO: Remove this from Braze template
-            "Post Code" -> "", //TODO: Remove this from Braze template
-            "Country" -> user.billingAddress.country.name,
-            "Date of first payment" -> formatDate(SubscriptionEmailFieldHelpers.firstPayment(paymentSchedule).date),
-            "Currency" -> currency.glyph,
-            "Trial period" -> "14", //TODO: depends on Promo code or zuora config
-            "Subscription details" -> SubscriptionEmailFieldHelpers.describe(paymentSchedule, billingPeriod, currency, promotion)
-          )
+          paymentFields(pm, directDebitMandateId) ++
+            directReaderFields(promotion, billingPeriod, user, currency, paymentSchedule)
         case None /*Corporate*/ => List(
           "Subscription details" -> "Group subscription"
         )
-      })
+      }
+
+    val fields = List(
+      "ZuoraSubscriberId" -> subscriptionNumber,
+      "EmailAddress" -> user.primaryEmailAddress,
+      "First Name" -> user.firstName,
+      "Last Name" -> user.lastName,
+    ) ++ fieldsForReaderType
 
-      override def payload: String = super.payload(user.primaryEmailAddress, if (paidSubPaymentData.isDefined) "digipack" else "digipack-corp")
+    val dataExtensionName = if (paidSubPaymentData.isDefined) "digipack" else "digipack-corp"
 
-      override def userId: Either[SfContactId, IdentityUserId] = Left(sfContactId)
+    EmailFields(fields, Left(sfContactId), user.primaryEmailAddress, dataExtensionName)
+  }
+
+  def paymentFields(paymentMethod: PaymentMethod, directDebitMandateId: Option[String]): Seq[(String, String)] =
+    paymentMethod match {
+      case dd: DirectDebitPaymentMethod => List(
+        "Account number" -> mask(dd.bankTransferAccountNumber),
+        "Sort Code" -> hyphenate(dd.bankCode),
+        "Account Name" -> dd.bankTransferAccountName,
+        "Default payment method" -> "Direct Debit",
+        "MandateID" -> directDebitMandateId.getOrElse("")
+      )
+      case dd: ClonedDirectDebitPaymentMethod => List(
+        "Sort Code" -> hyphenate(dd.bankCode),
+        "Account number" -> mask(dd.bankTransferAccountNumber),
+        "Account Name" -> dd.bankTransferAccountName,
+        "Default payment method" -> "Direct Debit",
+        "MandateID" -> dd.mandateId
+      )
+      case _: CreditCardReferenceTransaction => List("Default payment method" -> "Credit/Debit Card")
+      case _: PayPalReferenceTransaction => Seq("Default payment method" -> "PayPal")
     }
+
+  def directReaderFields(
+    promotion: Option[Promotion],
+    billingPeriod: BillingPeriod,
+    user: User,
+    currency: Currency,
+    paymentSchedule: PaymentSchedule
+  ): Seq[(String, String)] = {
+    List(
+      "Subscription term" -> billingPeriod.noun,
+      "Payment amount" -> SubscriptionEmailFieldHelpers.formatPrice(SubscriptionEmailFieldHelpers.firstPayment(paymentSchedule).amount),
+      "Address 1" -> "", //TODO: Remove this from Braze template
```

Comment:
Do we actually need to send it at all? I think Braze is quite tolerant of missing fields.

Reply:
maybe not but that code is just moved rather than edited - changing the actual fields sent is outside the scope of this PR